org.eclipse.dawnsci.analysis.api/src/org/eclipse/dawnsci/analysis/api/processing/IOperationBean.java

```diff
@@ -151,5 +151,12 @@
 	 * @param linkEntry
 	 */
 	default void setLinkParentEntry(boolean linkEntry){};
+	
+	/**
+	 * Set whether the raw data is going to be overwritten
+	 * 
+	 * @param overwrite
+	 */
+	default void setMonitorForOverwrite(boolean overwrite){};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)